tests/test_resources.py

```diff
@@ -155,6 +155,144 @@ def create_purchase():
             self.assertIsInstance(collection.charge_invoice, Invoice)
             self.assertEqual(purchase.billing_info_uuid, 'uniqueUuid')
 
+    def test_purchase_with_ramp_plan(self):
```

Comment:
Should a few of these test for `refundable_total_in_cents`?

Reply:
could you give me some example, not sure how it works that `refundable_total_in_cents `.

Reply:
Since `refundable_total_in_cents` is what's added, I was wondering if we have (perhaps elsewhere?) something that's testing serialization of that attribute.

Reply:
ok, on it